### ISSUE TITLE
[Hacktoberfest] Create a logo

### ISSUE THREAD

**Issue by TiagoDanin** [TiagoDanin/Hacktoberfest-Auto-Label#7]
File name: logo.png

[My other projects - #Hacktoberfest](https://github.com/search?q=label%3Ahacktoberfest+state%3Aopen+type%3Aissue+user%3ATiagoDanin&type=Issues)

**Comment by jimmiehansson** [comment]
This is definitely not my strong side, but I'm willing to give it a shot. And here it is, please don't laugh.
No hard feelings if this is a terrible attempt!
🌈 🦄 

![logo](https://user-images.githubusercontent.com/5203605/47255069-3de7f100-d46b-11e8-8f99-1c3cb9b0ce35.png)

**Comment by jimmiehansson** [comment]
Adding one with dark background to give an idea 
![logo-dark](https://user-images.githubusercontent.com/5203605/47255121-31b06380-d46c-11e8-9f6a-a68e910f9064.jpg)

**Comment by TiagoDanin** [comment]
I liked it. @jimmiehansson dark is beautiful.
- Remove the "2018"
- Use dimension 400 x 400

**Comment by jimmiehansson** [comment]
@TiagoDanin That's great! The unfortunate version is that my computer did a fresh nose-dive before I managed to save the PSD. But here's another take at it without the 2018! 👽 full size and 400x400

Added som bevel to make it more distinct in its depth, will this version still conqueror? 

![logo](https://user-images.githubusercontent.com/5203605/47259086-24639b00-d4a5-11e8-9561-32a5a3972310.png)
![logo_full](https://user-images.githubusercontent.com/5203605/47259088-36453e00-d4a5-11e8-8918-370c9a992889.jpg)
